flow/Trace.cpp

```diff
@@ -43,9 +43,8 @@
 #undef min
 #endif
 
-Reference<ITraceLogFormatter> createLogFormatter() {
-	auto f = FLOW_KNOBS->TRACE_FORMAT;
-	std::transform(f.begin(), f.end(), f.begin(), ::tolower);
+namespace {
```

Comment:
What was the reason for adding this namespace? If it's needed, I think I'd prefer to have its contents be indented.

Reply:
> Thanks @ajbeamon. Sorry about the whitespace problems. For the anonymous namespace I was following the advice here: https://google.github.io/styleguide/cppguide.html#Unnamed_Namespaces_and_Static_Variables

That seems fair, thanks for the pointer. Any thoughts on indenting the body of the namespace?

Reply:
I indented it. Done